lifelines/statistics.py

```diff
@@ -208,7 +208,7 @@ def chisq_test(U, degrees_freedom, alpha):
 
 
 def two_sided_z_test(Z, alpha):
-    p_value = 1 - max(stats.norm.cdf(Z), 1 - stats.norm.cdf(Z))
+    p_value = 1 - np.max(stats.norm.cdf(Z), 1 - stats.norm.cdf(Z))
```

Comment:
ooo console battle! 

But in the cases where we evaluate `max` against `.shape`, the array is only 2 elements.